zerver/models.py

```diff
@@ -1000,7 +1003,9 @@ def build_message_dict(
             timestamp         = datetime_to_timestamp(pub_date),
             gravatar_hash     = gravatar_hash(sender_email), # Deprecated June 2013
             avatar_url        = avatar_url,
-            client            = sending_client_name)
+            client            = sending_client_name,
+            is_me_message     = is_me_message
```

Comment:
add a trailing comma here (it makes life easier as we add things in the future to lists like this)